src/scripts/concatenate-docs.js

```diff
@@ -49,7 +49,16 @@ function getDirectoryRecursive(basePath) {
 			}
 
 		// write compound target file
-		const targetFile = fs.createWriteStream(targetFilePath);
+    const targetFile = fs.createWriteStream(targetFilePath);
+
+    targetFile.write(`---
```

Comment:
It this needed? If so, why?

Reply:
So this makes sure even the `printable.md` files have sort. Without title and contributors it fails to parse so i've added generic ones as they are harmless to printable files